content/en/docs/setup/additional-setup/sidecar-injection/index.md

```diff
@@ -321,5 +348,5 @@ Optionally, it may also be desirable to clean-up other resources that
 were modified in this task.
 
 {{< text bash >}}
-$ kubectl label namespace default istio-injection-
+$ kubectl label namespace default istio-injection=disabled --overwrite
```

Comment:
I think either of these work. Need to search the rest of the docs, but I think the standard is to remove the label and not set it to `disabled`